static/js/admin.js

```diff
@@ -670,6 +682,20 @@ function _setup_page() {
                         ui.report_success(i18n.t("Any user may now invite new users!"), invite_by_admins_only_status);
                     }
                 }
+                if (response_data.inline_image_preview !== undefined) {
+                    if (response_data.inline_image_preview) {
+                        ui.report_success(i18n.t("Inline image preview will be shown!"), inline_image_preview_status);
+                    } else {
+                        ui.report_success(i18n.t("Inline image preview will not be shown!"), inline_image_preview_status);
+                    }
+                }
+                if (response_data.inline_url_embed_preview !== undefined) {
+                    if (response_data.inline_url_embed_preview) {
+                        ui.report_success(i18n.t("Inline url embed preview will be shown!"), inline_url_embed_preview_status);
+                    } else {
+                        ui.report_success(i18n.t("Inline url embed preview will not be shown!"), inline_url_embed_preview_status);
```

Comment:
these should be plural, e.g. "Inline image previews will not be shown".  Might want to find a less technical way to explain it to users, e.g. "Show previews of uploaded and linked images." and "Show text and image previews of linked websites."